### ISSUE TITLE
Broken `insert` option

### ISSUE THREAD

**Issue by medfreeman** [faceyspacey/extract-css-chunks-webpack-plugin#264]
While using the `insert` option, i get the following error when the application tries to insert a chunk in the browser:

```
Uncaught (in promise) ReferenceError: insert is not defined
    at bootstrap:891
    at new Promise (<anonymous>)
    at Function.requireEnsure [as e] (bootstrap:861)
    at Function.fn.e (bootstrap:155)
    at load (UpgradeLicense.tsx:7)
    at Object.config.then (universalImport.js:8)
```

In fact, the user provided `insert` function is not injected into the bundle code.

Culprit is this line (from #226): https://github.com/faceyspacey/extract-css-chunks-webpack-plugin/commit/d41bafa3b442c0f36a8aa26fddfa5f158043a4e5#diff-1fdf421c05c1140f6d71444ea2b27638L385

The code responsible for injecting it has been forgotten in this PR.

In addition to this, this part of the feature isn't covered by #202 tests, but i imagine it's not really possible to test it in a more realistic fashion.

Culprit here: https://github.com/faceyspacey/extract-css-chunks-webpack-plugin/commit/6cd6510f7a86345df8656638aff46b8949da4402#diff-20c6b42869bc2385bcef4e1d9a7afa04R118-R124

**Comment by khitrenovich** [comment]
@ScriptedAlchemy Is this fix out already? We are running into the same error on upgrade to v4.7.5...

**Comment by adi518** [comment]
@khitrenovich I'd say yes, because it's corresponding PR was merged.

**Comment by khitrenovich** [comment]
> @khitrenovich I'd say yes, because its corresponding PR was merged.

Okay, so I just did some deep dive into that.

The PR was merged on May 30, while v4.7.5 (latest) was released on May 18 - 
```
~> npm view extract-css-chunks-webpack-plugin time --json | grep 4.7.5
  "4.7.5": "2020-05-18T17:30:15.836Z"
~> 
```

So apparently the fix is not released yet, and I've just opened #297 about it.

**Comment by adi518** [comment]
Great, I hope it gets published soon, because it takes like a minute to be done with it.
